app/utils/text.ts

```diff
@@ -0,0 +1,6 @@
+export function slugify(string: string, sep: string = '-'): string {
+    return string.toLowerCase()
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5730106)